vertica/datadog_checks/vertica/views.py

```diff
@@ -0,0 +1,154 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+# System tables:
+# https://www.vertica.com/docs/9.2.x/HTML/Content/Authoring/AdministratorsGuide/Monitoring/Vertica/UsingSystemTables.htm
+# https://www.vertica.com/docs/9.2.x/HTML/Content/Authoring/SQLReferenceManual/SystemTables/VerticaSystemTables.htm
+
+
+class View(object):
+    name = ''
+
+    def __hash__(self):
```

Comment:
I'm pretty sure those methods are not needed as we don't seem to instantiate them.